presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/Lookup.java

```diff
@@ -28,7 +30,7 @@
      * If the node is not a GroupReference, it returns the
      * argument as is.
      */
-    PlanNode resolve(PlanNode node);
+    List<PlanNode> resolve(PlanNode node);
```

Comment:
Instead of changing this method, I would deprecate it and add a new method `resolveGroup` which would be used to implement this method.